catalog/async/catalog-async-postingestplugin/src/test/groovy/org/codice/ddf/catalog/async/processingplugin/ProcessingPostIngestPluginTest.groovy

```diff
@@ -25,10 +26,12 @@ import org.codice.ddf.catalog.async.data.api.internal.ProcessCreateItem
 import org.codice.ddf.catalog.async.data.api.internal.ProcessDeleteItem
 import org.codice.ddf.catalog.async.data.api.internal.ProcessRequest
 import org.codice.ddf.catalog.async.data.api.internal.ProcessUpdateItem
+import org.codice.ddf.catalog.async.data.impl.LazyProcessResourceImpl
 import org.codice.ddf.catalog.async.processingframework.api.internal.ProcessingFramework
 import org.codice.ddf.catalog.async.processingplugin.ProcessingPostIngestPlugin
 import spock.lang.Specification
 
+import java.util.function.Supplier
 import java.util.concurrent.Callable
 
 class ProcessingPostIngestPluginTest extends Specification {
```

Comment:
Are there unit tests in `ProcessingPostIngestPluginTest.groovy` for when more Resources are added by the plugins (NITF overview and original, for example)? Is that even a situation that needs to be tested?